bin/nav

```diff
@@ -219,6 +251,27 @@ def c_config(_):
                  "{}".format('\n'.join(CONFIG_LOCATIONS)))
 
 
+def command_config_path(_):
```

Comment:
For lintings' sake I understand you need *something* here for the arg even if it isn't used, but if it was used, what would it be?

Reply:
Agree. According to the argparse API, this function _must_ take an argument, but the implementation doesn't need it. Renamed all `_` args to something meaningful but still prefixed by an `_` to make the linter ignore them